www/_config.yml

```diff
@@ -4,4 +4,6 @@ markdown: kramdown
 highlighter: true
 author:
   name: deen
+  email: dennis@felsing.org
```

Comment:
Do I really need to put my email in there? That will probably get me to receive more spam.

Reply:
It's fine without, I just noticed it was an empty tag in the feed before. I'll remove it and change author to `DDNet Team`.